ui/lib/core/addon/templates/components/replication-dashboard.hbs

```diff
@@ -9,23 +9,15 @@
   {{/if}}
 
   <div class="selectable-card-container has-top-margin-xl">
-    {{yield (hash 
+    {{yield (hash
```

Comment:
not sure why my spaces are not getting removed.  I used to have my ide setup to do that.  I'll look into that.